public/conduct.jade

```diff
@@ -0,0 +1,30 @@
+div.jsla-toplevel-container.jsla-content-page
```

Comment:
@jcblw isn't there a way to create redirect aliases? Or do we just have to duplicate the page like this?